nixos/modules/services/security/proxychains.nix

```diff
@@ -0,0 +1,184 @@
+{ config, lib, pkgs, ... }:
+with lib;
+let
+
+  cfg = config.services.proxychains;
+
+  configFile = ''
+    ${cfg.chain.type}
+    ${optionalString (cfg.chain.type == "random")
+    "chain_len = ${builtins.toString cfg.chain.length}"}
+    ${optionalString cfg.proxyDNS "proxy_dns"}
+    ${optionalString cfg.quietMode "quiet_mode"}
+    ${cfg.remoteDNSSubnet}
+    ${cfg.tcpReadTimeOut}
+    ${cfg.tcpConnectTimeOut}
+    ${cfg.localnet}
+    ${cfg.proxies}
+    ${cfg.extraConfig}
```

Comment:
Does it make sense to have an `extraConfig` at the end? From looking at the config file, it doesn't seem like anything other than proxies can go after the `[ProxyList]` section.